packages/localize/hooks/converter.ios.ts

```diff
@@ -0,0 +1,98 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as plist from "simple-plist";
+
+import { ConverterCommon } from "./converter.common";
+import { DataProvider, I18nEntries, Languages } from "./data.provider";
+import { encodeKey, encodeValue } from "../resource.ios";
+
+// export class ConverterIOS extends ConverterCommon {
+export class Converter extends ConverterCommon {
+  protected cleanObsoleteResourcesFiles(resourcesDirectory: string, languages: Languages): this {
+    fs.readdirSync(resourcesDirectory).filter(fileName => {
+      const match = /^(.+)\.lproj$/.exec(fileName);
+      return match && !languages.has(match[1]);
+    }).map(fileName => {
+      return path.join(resourcesDirectory, fileName);
+    }).filter(filePath => {
+      return fs.statSync(filePath).isDirectory();
+    }).forEach(lngResourcesDir => {
+      ["InfoPlist.strings", "Localizable.strings"].forEach(fileName => {
+        const resourceFilePath = path.join(lngResourcesDir, fileName);
+        this.removeFileIfExists(resourceFilePath);
+      });
+      this.removeDirectoryIfEmpty(lngResourcesDir);
+    });
+    return this;
+  }
+
+  protected createLanguageResourcesFiles(
+    language: string,
+    isDefaultLanguage: boolean,
+    i18nEntries: I18nEntries
+  ): this {
+    const infoPlistStrings: I18nEntries = new Map();
+    i18nEntries.forEach((value, key) => {
+      if (key === "app.name") {
+        infoPlistStrings.set("CFBundleDisplayName", value);
+        infoPlistStrings.set("CFBundleName", value);
```

Comment:
This should be left alone, otherwise it overwrites the NS-expected value of `${PRODUCT_NAME}` and causes the CLI to not find the build output .ipa.

Reply:
Thanks @facetious - we published a @nativescript/localize 5.0.0 version. If you get a chance to try it. We make this adjustment to remove this condition if seems for sure should be removed and publish a 5.0.1 patch early this week to finish things off here.

Reply:
@facetious - 5.0.1 published now which removed that app.name hook section.